.github/workflows/orca.yml

```diff
@@ -61,18 +61,18 @@ jobs:
             php-version: "8.0"
             coveralls-enable: "FALSE"
             orca-version: "^3"
-          # - orca-job: ISOLATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_DEV
-          #   php-version: "8.1"
-          #   coveralls-enable: "FALSE"
-          # - orca-job: INTEGRATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_DEV
-          #   php-version: "8.1"
-          #   coveralls-enable: "FALSE"
-          # - orca-job: ISOLATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_BETA_OR_LATER
-          #   php-version: "8.1"
-          #   coveralls-enable: "FALSE"
-          # - orca-job: INTEGRATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_BETA_OR_LATER
-          #   php-version: "8.1"
-          #   coveralls-enable: "FALSE"
+
+          # Testing Drupal 11 in php 8.3.
+          - orca-job: ISOLATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_DEV
+            php-version: "8.3"
+          - orca-job: INTEGRATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_DEV
+            php-version: "8.3"
+            -enable: "FALSE"
```

Comment:
It looks like this used to be `coveralls-enable` and got cut off. I notice the other instances of `coveralls-enable` have been removed. I assume they were all supposed to have been kept intact.

Reply:
yes it was `coveralls-enable`, fixed it